packages/react-error-overlay/src/__tests__/get-source-map.js

```diff
@@ -3,6 +3,8 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ * 
+ * @jest-environment jsdom
```

Comment:
Hi , as per the new jest 27 the default jest-environment has changed to "node"
but also this was causing tests to fail. but since its already specified . then maybe its not needed

https://github.com/facebook/create-react-app/pull/10748#issuecomment-807595634
https://jestjs.io/blog/2021/05/25/jest-27